ci/security/pipeline.template.yml

```diff
@@ -14,3 +14,8 @@ steps:
     timeout_in_minutes: 5
     agents:
       queue: linux
+
+  - command: bin/ci-builder run stable cargo vet check
```

Comment:
This means QA will be the ones that get pinged when a new Rust dependency has been introduced. What are we supposed to do with that? Previously it was difficult for me to judge what to do, we'd probably just end up allowlisting them or asking the dev. So I'd suggest to put this in ci/tests/pipeline.template.yml.

The main worry would be if this could break without a PR, thus making main red. But if my understanding of cargo vet is correct this shouldn't happen.